Name: Add Date dd-MMM-yy
Version: 1.3.5.2-1
Author: AnthonyK
Description: Adds date to the top bar next to the time in format dd-MMM-yy. (e.g 13-Jan-10)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -2552,7 +2552,8 @@
 		}
 	}
 	format = format.replace(/\s?a\s?/,"");	
-	$('clock').innerHTML = Mojo.Format.formatDate(date, {time: format});				
+	var formatDate = Mojo.Format.formatDate(date, 'dd-MMM-yy');
+	$('clock').innerHTML = formatDate+'&nbsp;&nbsp;&nbsp;&nbsp;'+Mojo.Format.formatDate(date, {time:format});
 	$('today').innerHTML = Mojo.Format.formatDate(date,{date:'short'});
 },
 
